src/dragdrop.js

```diff
@@ -0,0 +1,56 @@
+import { save, load, compare } from './data.js';
```

Comment:
Good working on trying to stay DRY with your code.

Reply:
Thanks  for the review, I will implement the changes.